prime-router/metadata/hl7_mapping/ORU_R01/base/specimen.yml

```diff
@@ -47,16 +47,25 @@ elements:
     schema: ../../common/datatype/ce-coded-with-exceptions
 
   - name: specimen-type-modifier
-    value: [ '""' ]
-    hl7Spec: [ '%{hl7SpecimenFieldPath}-5' ]
+    resource: '%resource.extension("https://reportstream.cdc.gov/fhir/StructureDefinition/specimen-type-modifier").value'
```

Comment:
When getting extensions, you can re-use the constant for the RS extension URL defined at the top of `ORU_R01-base.yml` and get the value this way `.extension(%`rsext-specimen-type-modifier`).value`